### DISCUSSION TITLE
Made some type 0 wallpapers for popular operating systems

### DISCUSSION THREAD

**Discussion by uttsav1025** [daltonmenezes/aura-theme#130] [General]
Hello!

The [README](https://github.com/daltonmenezes/aura-theme#contributing)  instructed to ask before working on a pull-request, so
here are two of the nine wallpapers I created based on the [type 0 wallpapers](https://github.com/daltonmenezes/aura-theme/tree/main/packages/wallpapers#type-0)  :

https://github.com/uttsav1025/wallpapers-for-aura-theme

![debian](https://user-images.githubusercontent.com/74333675/143390457-6bad5f3a-2a0b-4697-bb17-d87eb22fefc7.png)

![windows11](https://user-images.githubusercontent.com/74333675/143390495-42f7642f-78f9-4865-9400-27b875e30313.png)

**Comment by daltonmenezes** [comment]
Hi @uttsav1025 , I'm sorry for the delay. I'm wondering what's the best approach to make the wallpapers page not be a mess and hard to load with a lot of images as it grows, maybe creating a Community section for the wallpapers made by the community tagging only the **user/repo** link? Any ideas?

**Reply by kayozxo** [reply]
Maybe by displaying low-res images in tabular form (GitHub readme), and pointing to high-res images hosted elsewhere, like Imgur, or maybe making a GitHub static site and lazy loading them.
